.azure-pipelines-v8.yml

```diff
@@ -18,9 +18,11 @@ jobs:
           sudo apt update
           sudo apt install -y pkg-config
         displayName: "Install dependencies"
+        continueOnError: false
```

Comment:
Isn't that the default?

Reply:
apparently not, or at least it was continuing when it should have failed and I put that to make it stop.

Could it be some global config that is overriding it?

Reply:
Very weird, https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#job alleges that it defaults to false. I can't find any information about an override, if there is, we're certainly not setting it on purpose.